src/features/headline/redux/middleware.ts

```diff
@@ -8,7 +8,7 @@ import {
 import { UPDATE_SEARCH_QUERY } from "./constants";
 import { SocketService } from "../../../services/socketService";
 import { Middleware, AnyAction } from "redux";
-import { HeadlineState } from "./reducer";
+import { AppState } from "../../../store";
```

Comment:
tip: You can remove the ../../../ by adding `'baseUrl': './src'` to your tsconfig compiler options